loops-and-arrays.js

```diff
@@ -6,13 +6,17 @@
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+for (let i = 0; i < names.length; i++) {
+  names[i] = names[i] + "je";
+  console.log(names[i]);
+}
+
```

Comment:
Prima stukje code. Alleen geeft hij de namen onder elkaar weer omdat hij ze een voor een looped. En in de verwachte uitkomst stond het naast elkaar: ["Henkje", "Pietje", "Fredje", "Joopje"].

Dit is makkelijk op te lossen door de console.log buiten de for loop te plaatsen en console.log(names) te loggen ipv console.log(names[i]).

Maar dit vind ik eigenlijk maar een schoonheidsfoutje de code zelf ziet er prima uit op deze manier.